### PR TITLE
Feature weather - For Review

### PR DESCRIPTION
**Pull request by paulo-techie**
- Set up an HTML document with the appropriate links to the JavaScript and CSS files for the project
- Wrote the functions that hit the API. The functions take a location and return the weather data for that location. 
- The location is submitted through an input form.
- Display the location's weather information on the web page
- Function processes the JSON data from the API and returns an object with only the data required for the app.
- Added some styling through CSS.

### CODE REVIEW

**Comment by mke2111** [comment]
## STATUS: INVALID 🔴 

### Hey Paul, 👋🏽  
I'm invalidating this PR simply because the linters are failing. Fix all the detected errors and submit another CRR. 

>If you need extra help with this, don't hesitate to reach out to me.

File: src/geo-api.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function currentWeatherLocation() {
```
Comment (addod19) [inline src/geo-api.js]: Kindly use the ES6 arrow syntax instead

File: src/weather-api.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const API_KEY = '2745510d199eb84e68a506d68e71467a';
+
+function loadWeather(city) {
```
Comment (addod19) [inline src/weather-api.js]: Kindly use the ES6 arrow syntax instead

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @paulo-techie, congratulations on this project :+1: You followed  Git Flow, where PR comes from the feature to the development branch :smile: All linter checks passed :muscle: README is professional :fireworks: Code is structured well :100: 
However, there are a few comments we need to address, kindly check the comments below

- Kindy use the right linter file. You can verify the recommended file [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)
- Webpack is not configured well, hence we are missing the webpack.config file. Let us fix it
- The live version of the app is not working as expected, let us kindly fix it
- We are missing the toggle functionality where can toggle between the Celcius and Fahrenheit
- We are also missing the functionality where the appearance changes based on the data received from the API. Let us kindly address this
- Kindly use the ES6 arrow syntax instead of the function keyword

[Optional]
- It would be nice if we implement the loading functionality to load until data is retrieved from the API

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The app runs just fine. 👍🏽 

### **_Room for Improvement_** 🤪 

- [ ]  When the search city entered is not found your app does not crash but it doesn't provide users with any feedback on the result. Add a modal to inform users about this.

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**